modules/tinymce/src/core/test/ts/browser/fmt/RemoveFormatTest.ts

```diff
@@ -29,6 +30,10 @@ describe('browser.tinymce.core.fmt.RemoveFormatTest', () => {
     editor.formatter.unregister('format');
   };
 
+  const getContent = (editor: Editor) => {
```

Comment:
You can configure tinymce to not indent the source by setting indent: false in the bddSetupLight call.

Reply:
nice to know, updated